src/screens/IndexScreen.js

```diff
@@ -528,6 +539,13 @@ export const IndexScreen = ({ navigation, route }) => {
                           <RegularText>{categoryListIntroText}</RegularText>
                         </Wrapper>
                       )}
+                      {query === QUERY_TYPES.POINTS_OF_INTEREST &&
+                        !!htmlContentName &&
+                        !!htmlContent && (
+                          <Wrapper>
+                            <HtmlView html={htmlContent} />
+                          </Wrapper>
+                        )}
```

Comment:
these are too much checks. please only regard the part what is rendered. we want to render `htmlContent`, so `htmlContent` needs to be defined. the other checks do not matter here. in the render part we try to avoid as much logics as possible. at this point all logics have to be done, to result in a defined or undefined `htmlContent`